LICENCE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 
+Copyright (c) 2015 Shanty
```

Comment:
Best to put our names here for now, as Shanty isn't a company or anything we can prove ownership of.